src/styles/login.scss

```diff
@@ -0,0 +1,6 @@
+$color: rgb(255,0,0);
+
+.full-screen-container {
+    height: 100%;
+    width: 100%;
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/nezago/phone-history-store-frontend/pullRequest?prid=6016116)